textproc/lilypond/Portfile

```diff
@@ -63,16 +83,27 @@ depends_build-append \
                     port:p5.28-pod-simple \
                     port:p5.28-podlators \
                     port:p5.28-scalar-list-utils \
-                    port:netpbm \
                     port:pkgconfig \
                     port:t1utils \
                     port:texinfo \
+                    port:urw-core35-fonts
+
+if {${subport} eq "${name}"} {
```

Comment:
You don't need quotation marks around `${name}` here or anywhere else.

Reply:
Fixed.